ais-infra-alerting/template.yaml

```diff
@@ -0,0 +1,393 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+
+Description: >-
+  This stack sets up the resources required for AIS alerting
+
+Parameters:
+  Environment:
+    Description: The name of the environment to deploy to
+    Type: String
+    AllowedValues:
+      - "dev"
+      - "build"
+      - "staging"
+      - "integration"
+      - "production"
+  ProductTagValue:
+    Description: Value for the Product Tag
+    Type: String
+    Default: GOV.UK One Login
+  OwnerTagValue:
+    Description: Value for the Owner Tag
+    Type: String
+    Default: PLACEHOLDER
+  SourceTagValue:
+    Description: Value for the Source Tag
+    Type: String
+    Default: govuk-one-login/ais-infra/ais-infra-alerting/template.yaml
+  SystemTagValue:
+    Description: Value for the System Tag
+    Type: String
+    Default: Account Interventions Service
+  SlackWorkspaceId:
+    Description: >
+      The ID of the Slack workspace where alert notification messages are
+      posted. This is retrieved from the AWS Chatbot integration.
+    Type: String
+    Default: "T8GT9416G"
+    AllowedPattern: "\\w+"
+    ConstraintDescription: "must be an AWS Chatbot Slack workspace ID"
+  SlackChannelId:
+    Description: >
+      The ID of the Slack channel where alert notification messages are posted.
+      This is taken from the channel details in Slack.
+      Default is the di-accounts-bravo-ais-notifications channel.
+    Type: String
+    Default: "C065MT3RKV4"
```

Comment:
nit: could we just leave a comment stating the name of the slack channel please :)